nodo-installer.sh

```diff
@@ -90,7 +90,7 @@ sleep 3
 swapfile=/media/monero/swap
 sleep 1
 showtext "Setting up swap on $swapfile"
-dd if=/dev/zero of="$swapfile" bs=1M count=1024 conv=sync
+dd if=/dev/zero of="$swapfile" bs=1M count=10240
 mkswap "$swapfile"
```

Comment:
`mkswap` doesnt work for me on desktop w/o sudo (even if using root account). Not sure if an issue here